aten/src/ATen/functorch/ADInterpreters.cpp

```diff
@@ -44,12 +44,17 @@ Tensor materializeGradWrappers(const Tensor& tensor, int64_t current_level) {
   return makeTensorWrapper(tensor, current_level, /*is_immutable=*/true);
 }
 
+static Tensor base_lift(const Tensor& tensor) {
+  auto tensor_ = unwrapIfDead(tensor);
+  return materializeGradWrappers(tensor_, level());
+}
```

Comment:
This is a bug-fix. The equivalent of Interpreter.lift in C++ unwraps dead tensors and then materializes grad wrappers.